frontend/components/wiki/Breadcrumbs.jsx

```diff
@@ -0,0 +1,66 @@
+import Link from "next/link";
+import PropTypes from "prop-types";
+import React from "react";
+import { useRouter } from "next/router";
+
+export default function Breadcrumbs(props) {
+  const { pathname } = useRouter();
+  const currentPage = pathname.slice(6);
+  const breadcrumbPath = currentPage.split("/");
+
+  const breadcrumbArray = [];
+
+  function BreadcrumbItem({ listItem }) {
+    return (
+      <React.Fragment>
+        <span className="">&#8250;</span>
+        <li className="flex">
+          <Link href={"/wiki/" + listItem.url.replace(/\index.mdx$/, "").replace(/\.mdx$/, "")}>
+            {listItem.name.replace(/\.mdx$/, "")}
+          </Link>
+        </li>
+      </React.Fragment>
+    );
+  }
+  BreadcrumbItem.propTypes = {
+    listItem: PropTypes.shape({
+      name: PropTypes.string,
+      url: PropTypes.string,
+    }),
+  };
```

Comment:
Can `BreadcrumbItem` be moved outside of `Breadcrumbs`? It could remain in the same file and not be exported, but I expect React sees it as a new component and re-renders it every time.

[Avoid Declaring React Component Inside Parent Component](https://levelup.gitconnected.com/code-review-avoid-declaring-react-component-inside-parent-component-1768a645f523)

Reply:
@antw , thank you for reviewing and providing an improvement. I have changed the code, so `breadcrumbItem `is outside of  `Breadcrumbs`